### PR TITLE
Function to get an object of the vector's metrics

### LINKED ISSUE #20: Function to get an object of the vector's metrics
Currently we split the vector in the getScore() function and create an object with the abbreviations as key and the vector values as value:
https://github.com/turingpointde/cvss.js/blob/main/lib/cvss.js#L16-L22

We should refactor this into a separate function and make it available to the public API.

It is useful to provide this function to the user. In the future it is imaginable that even more information will be generated by this function.

### PR DESCRIPTION
**Pull request by vivek32ta**
# Description

Refactored a separate function `getVectorObject()` with respective test cases and made it publicly available. Added documentation as well.

Fixes #20 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: test/cvss.spec.js [hunk 1]
```diff
@@ -49,3 +49,37 @@ describe("Rating Tests", () => {
     expect(vector.getRating()).toBe("Critical");
   });
 });
+
+describe("Rating Tests", () => {
```
Comment (Fubinator) [inline test/cvss.spec.js:53]: Could you please change the name of the test to something like `Vector Object Tests`?

Comment (vivek32ta) [reply test/cvss.spec.js]: Oh damn I had changed that... Good catch. It must've changed while I was undoing something hahahah.
I'm so sorry. I'll do it right away!

Thanks

Comment (Fubinator) [reply test/cvss.spec.js]: No worries!

**Review by Fubinator — APPROVED** [review]

**Comment by vivek32ta** [comment]
Thank you for helping @Fubinator. I really appreciate it!

It was absolutely wonderful contributing here  :")

Best,
Vivek

**Comment by Fubinator** [comment]
You're welcome! And thank you for contributing! @vivek32ta

**Pull request merged**